tests/Unit/Evolution/DiscontinuousGalerkin/Limiters/Test_LimiterActionsWithMinmod.cpp

```diff
@@ -66,13 +70,19 @@ struct component {
   using chare_type = ActionTesting::MockArrayChare;
   using array_index = ElementIndex<Dim>;
   using const_global_cache_tags = tmpl::list<LimiterTag>;
-  using simple_tags =
-      db::AddSimpleTags<TemporalId, domain::Tags::Mesh<Dim>,
-                        domain::Tags::Element<Dim>,
-                        domain::Tags::ElementMap<Dim>,
-                        domain::Tags::Coordinates<Dim, Frame::Logical>,
-                        domain::Tags::Coordinates<Dim, Frame::Inertial>, Var>;
-  using compute_tags = db::AddComputeTags<domain::Tags::SizeOfElement<Dim>>;
+  using simple_tags = db::AddSimpleTags<
+      TemporalId, domain::Tags::Mesh<Dim>, domain::Tags::Element<Dim>,
+      domain::Tags::ElementMap<Dim, Frame::Grid>,
+      domain::CoordinateMaps::Tags::CoordinateMap<2, Frame::Grid,
+                                                  Frame::Inertial>,
+      ::Tags::Time, domain::Tags::FunctionsOfTime, Var>;
+  using compute_tags =
+      db::AddComputeTags<::domain::Tags::LogicalCoordinates<Dim>,
```

Comment:
For my understanding: why are the logical coordinates a compute tag? is it with an eye to the future and AMR, so they auto-update on domain changes?

Reply:
Yes, and because they depend on the mesh (so Gauss vs. Gauss-Lobatto and Legendre vs. Chebyshev basis).